crates/matrix-sdk/src/widget/machine/driver_req.rs

```diff
@@ -219,9 +223,11 @@ pub(crate) struct SendEventRequest {
     pub(crate) state_key: Option<String>,
     /// Raw content of an event.
     pub(crate) content: Box<RawJsonValue>,
-    /// Additional send event parameters to send a future event.  
-    #[serde(flatten)]
-    pub(crate) future_event_parameters: Option<FutureParameters>,
+    /// The delay for this request in ms.
+    /// The widget does not use the `org.matrix.msc4140.delay` unstable prefix.
+    /// It uses just an optional `delay` property. If provided the response will
+    /// contain a delay_id instead of a event_id.
```

Comment:
Instead of explaining that the unstable prefix isn't used, maybe just reference the MSC
```suggestion
    /// The optional delay (in ms) to send the event at.
    /// If provided, the response will contain a delay_id instead of a event_id.
    /// Defined by [MSC4157](https://github.com/matrix-org/matrix-spec-proposals/pull/4157)
```